db/schema.rb

```diff
@@ -11,10 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20151103161642) do
+ActiveRecord::Schema.define(version: 20160205150033) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "pg_stat_statements"
```

Comment:
Ha. This line has leaked in. I imagine it'll leak back out again at some point. So lets just leave it.